yabt/buildcontext.py

```diff
@@ -23,6 +23,7 @@
 
 
 from collections import defaultdict, deque
+from colorama import Fore, Style
```

Comment:
colorama is 3rd party. should be grouped with networkx

Reply:
Done.